### DISCUSSION TITLE
Lightcontroller

### DISCUSSION THREAD

**Discussion by RonXTCdaBass** [rocksdanister/lively#1887] [Show and tell]
Hello all!

i wanna share my actual wallpaper.
nice as background for party ;) 
...not perfect yet, but my ^^

![screenshot](https://ronxtcdabass.lima-city.de/frameset/down/-=Light-Controller=-.jpg)

[DEMO](https://ronxtcdabass.lima-city.de/frameset/lightjockey/lively.html) - [Download](https://ronxtcdabass.lima-city.de/frameset/down/-=Light-Controller=-.zip)

**Comment by BobtitoMc** [comment]
does not work00

**Reply by RonXTCdaBass** [reply]
strange... it works for me. 
Win10 and Lively v2.2.1.0... 
just a little silly, e.g. until the part accepts the mouse clicks I have to click a few times, but then everything works out
